### DISCUSSION TITLE
Getting issue to clone Kohya

### DISCUSSION THREAD

**Discussion by leonviz87** [kohya-ss/sd-scripts#1422] [Q&A]
Hi guys i keep facing this issue when trying to clone the kohya ss, not too sure if anyone can help? 

![image](https://github.com/kohya-ss/sd-scripts/assets/133586245/62b33d0f-4710-4dae-b5bb-83753a9455ac)

**Comment by kohya-ss** [comment]
The log said `Python was not found`, so please install Python first.

**Reply by leonviz87** [reply]
Hmm but i did install python from the installer that i have downloaded though, how do i check if i install correctly?
